resources/build/trigger-build-bot.inc.sh

```diff
@@ -88,7 +88,9 @@ function build_bot_check_messages() {
     # parse without risking escaping our bash jail
 
     if [[ ! -z "${buildBotCommand// }" ]]; then
-      build_bot_update_commands $buildBotCommand
+      local build_bot_args
+      read -r -a build_bot_args <<< "${buildBotCommand}"
+      build_bot_update_commands "${build_bot_args[@]}"
```

Comment:
I'm struggling to understand what is fixed here -- `$buildBotCommand` was a space separated string already so why does this work when the previous pattern didn't?

Reply:
Eeek!